internal/tasks/device_render.go

```diff
@@ -66,29 +70,54 @@ func (t *DeviceRenderLogic) RenderDevice(ctx context.Context) error {
 	emptyIgnitionConfig, _, _ := config_latest.ParseCompatibleVersion([]byte("{\"ignition\": {\"version\": \"3.4.0\"}"))
 	t.renderedConfig = &emptyIgnitionConfig
 
+	invalidConfigs := []string{}
 	if device.Spec != nil && device.Spec.Config != nil {
 		for i := range *device.Spec.Config {
 			configItem := (*device.Spec.Config)[i]
-			err := t.handleConfigItem(ctx, &configItem)
+			name, err := t.handleConfigItem(ctx, &configItem)
 			if err != nil {
-				return t.setStatus(ctx, err)
+				invalidConfigs = append(invalidConfigs, name)
```

Comment:
We don't want to lose the original error 
```suggestion
if err != nil {
    if errors.Is(err, ErrUnknownName) {
	invalidConfigs = append(invalidConfigs, unknownName)
   }
   //log/handle original error
 }
```